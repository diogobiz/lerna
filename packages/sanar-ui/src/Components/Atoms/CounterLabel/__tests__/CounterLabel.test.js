
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESCounterLabel from '../'

it('renders correctly', () => {
    const component = (
        <ESCounterLabel>
            <h1>ESCounterLabel</h1>
        </ESCounterLabel>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
