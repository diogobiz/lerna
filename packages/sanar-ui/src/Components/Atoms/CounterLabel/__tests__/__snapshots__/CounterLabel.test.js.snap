// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  aria-label={null}
  className="ant-typography"
  style={
    Object {
      "WebkitLineClamp": null,
      "fontSize": 12,
    }
  }
>
  <span
    aria-label={null}
    className="ant-typography"
    style={
      Object {
        "WebkitLineClamp": null,
      }
    }
  >
    <strong>
      
    </strong>
  </span>
   undefineds
</span>
`;
